cleanlab/object_detection/summary.py

```diff
@@ -34,6 +43,412 @@
+
+    for c in class_metrics.keys():
+        acc_score = class_metrics[c]["TP"] / (
+            class_metrics[c]["TP"] + class_metrics[c]["FP"] + class_metrics[c]["FN"]
+        )
+        class_metrics[c]["accuracy"] = round(acc_score, 2)
+
+    return class_metrics
+
+
+def get_sorted_bbox_count_idxs(labels, predictions):
+    """
+    Returns a tuple of idxs and bounding box counts of images sorted from highest to lowest number of bounding boxes.
+
+    This plot can help you discover images with abnormally many/few object annotations.
+
+    Parameters
+    ----------
+    labels:
+        Annotated boxes and class labels in the original dataset, which may contain some errors.
+        Refer to documentation for this argument in :py:func:`object_counts_per_image <cleanlab.object_detection.summary.object_counts_per_image>` for further details.
+
+    predictions:
+        Predictions output by a trained object detection model.
+        Refer to documentation for this argument in :py:func:`object_counts_per_image <cleanlab.object_detection.summary.object_counts_per_image>` for further details.
+
+
+    Returns
+    -------
+    sorted_idxs: List[Tuple[int, int]], List[Tuple[int, int]]
+        A tuple containing two lists. The first is an array of shape ``(N,)`` containing the number of annotated objects for each image in the dataset.
+        The second is an array of shape ``(N,)`` containing the number of predicted objects for each image in the dataset.
+    """
+    lab_count, pred_count = object_counts_per_image(labels, predictions)
+    lab_grouped = zip([i for i in range(len(labels))], lab_count)
+    pred_grouped = zip([i for i in range(len(predictions))], pred_count)
+
+    sorted_lab = sorted(lab_grouped, key=lambda x: x[1], reverse=True)
+    sorted_pred = sorted(pred_grouped, key=lambda x: x[1], reverse=True)
+
+    return sorted_lab, sorted_pred
+
+
+def plot_class_size_distributions(
+    labels, predictions, class_names=None, class_to_show=MAX_CLASS_TO_SHOW
+):
+    """
+    Plots the size distributions for bounding boxes for each class.
+
+    This plot can help you find annotated/predicted boxes for a particular class that are abnormally big/small.
+
+    Parameters
+    ----------
+    labels:
+        Annotated boxes and class labels in the original dataset, which may contain some errors.
+        Refer to documentation for this argument in :py:func:`object_counts_per_image <cleanlab.object_detection.summary.object_counts_per_image>` for further details.
+
+    predictions:
+        Predictions output by a trained object detection model.
+        Refer to documentation for this argument in :py:func:`object_counts_per_image <cleanlab.object_detection.summary.object_counts_per_image>` for further details.
+
+    class_names: optional
+        Optional dictionary mapping one-hot-encoded class labels back to their original class names in the format ``{"integer-label": "original-class-name"}``.
+
+    class_to_show: optional
```

Comment:
If we have a MAX classes to show, then class distributions should be ordered by most common occurring classes first